src/server/ua_services_nodemanagement.c

```diff
@@ -665,6 +669,26 @@ copyChild(UA_Server *server, UA_Session *session,
             return retval;
         }
 
+        /* Add HasInterface references to the child */
+        if (rd->nodeClass == UA_NODECLASS_OBJECT && !UA_NodeId_isNull(&rd->typeDefinition.nodeId)) {
+            UA_Node *typeNode = NULL;
+            retval = UA_NODESTORE_GETCOPY(server, &rd->typeDefinition.nodeId, &typeNode);
```

Comment:
It would be better to not get a copy, but a const reference to the original node.